### PR TITLE
[DE] Add `Tournaments/Badge-weighted seeding`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
This article was pretty difficult to translate. I hope my translation is comprehensible.

Notes on translations:

- `to seed` - `Turniereinstufung auslosen`
- `seed` - `Platzierung`, `Seed`

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/Badge-weighted_seeding/de.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+- `Anzahl Abzeichen` (≥ 0): die Anzahl an Abzeichen auf dem Profil eines Spielers, die als Preise für vergangene Turniere desselben [Spielmodus](/wiki/Game_mode) gewonnen wurden
+- `Globaler Rang` (≥ 1): der [Performance-Punkte-Rang](/wiki/Ranking#performance-points-ranking) auf der globalen Rangliste gemäß dem [Spielmodus](/wiki/Game_mode) des Turniers
+
+Diese Funktion erfüllt Eigenschaften, die für den Zweck von BWS wesentlich sind:
+
+- Der Seed eines Spielers sinkt in größeren Schritten, als die Spieler weitere Abzeichen erlangen können, oder mit anderen Worten, jedes neue Abzeichen ist mehr Wert als das vorherige.[^hippo-cup-bws] Beachte, dass dies in der obigen Formel nur bis zu einem Punkt zwischen 2 und 6 Abzeichen zutrifft (abhängig von `Globaler Rang`). Danach tritt ein Wendepunkt ein und der Effekt kehrt sich ins Gegenteil um.
+- BWS-Seeds sind immer kleiner als oder gleich `Globaler Rang`. Sie sind gleich, wenn der Spieler keine Abzeichen besitzt.
+- BWS-Seeds haben dieselbe Größenordnung wie `Globaler Rang`.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/Badge-weighted_seeding/de.md:38]: I prefer the current version, though here are the alternatives I had noted:

- `Verteilung`
- `Spektrum`
- `Bereich`

File: wiki/Tournaments/Badge-weighted_seeding/de.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```
+Seed = Globaler Rang ^ (0.9937 ^ (Anzahl Abzeichen ^ 2))
+```
+
+- `Anzahl Abzeichen` (≥ 0): die Anzahl an Abzeichen auf dem Profil eines Spielers, die als Preise für vergangene Turniere desselben [Spielmodus](/wiki/Game_mode) gewonnen wurden
+- `Globaler Rang` (≥ 1): der [Performance-Punkte-Rang](/wiki/Ranking#performance-points-ranking) auf der globalen Rangliste gemäß dem [Spielmodus](/wiki/Game_mode) des Turniers
+
+Diese Funktion erfüllt Eigenschaften, die für den Zweck von BWS wesentlich sind:
+
+- Der Seed eines Spielers sinkt in größeren Schritten, als die Spieler weitere Abzeichen erlangen können, oder mit anderen Worten, jedes neue Abzeichen ist mehr Wert als das vorherige.[^hippo-cup-bws] Beachte, dass dies in der obigen Formel nur bis zu einem Punkt zwischen 2 und 6 Abzeichen zutrifft (abhängig von `Globaler Rang`). Danach tritt ein Wendepunkt ein und der Effekt kehrt sich ins Gegenteil um.
```
Comment (respektive) [inline wiki/Tournaments/Badge-weighted_seeding/de.md:36]:
```suggestion
- Der Seed eines Spielers sinkt in größeren Schritten, je mehr Abzeichen er erlangt, oder mit anderen Worten, jedes neue Abzeichen ist mehr Wert als das vorherige.[^hippo-cup-bws] Beachte, dass dies in der obigen Formel nur bis zu einem Punkt zwischen 2 und 6 Abzeichen zutrifft (abhängig von `Globaler Rang`). Danach tritt ein Wendepunkt ein und der Effekt kehrt sich ins Gegenteil um.
```

File: wiki/Tournaments/Badge-weighted_seeding/de.md (hunk 1 above)
Comment (respektive) [reply wiki/Tournaments/Badge-weighted_seeding/de.md:38]: seems fine to me

File: wiki/Tournaments/Badge-weighted_seeding/de.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- Abzeichen, die für Turniere mit Rangbeschränkung verliehen werden, wirken sich weiterhin auf die BWS-Seeds der Spieler aus, auch wenn ihre Ränge außerhalb dieser Rangbereiche liegen.[^badge-appeals] Im Allgemeinen kann es als Nachteil angesehen werden, dass trotz der vielen verschiedenen Faktoren zwischen Turnieren, die dazu führen, dass Abzeichenpreise mehr oder weniger schwierig zu gewinnen sind, alle Abzeichen gleich behandelt werden.
+  - Einige experimentelle Versionen von BWS beinhalten Mechanismen, um dieses Problem zu lösen.[^oet-bws]
+- Für das Turnierpersonal ist zusätzliche Arbeit erforderlich, um festzustellen, welche Abzeichen für das Turnier relevant sind, und um den Seed zu berechnen.
+
+::{ flag=US }:: [DigitalHypno](https://osu.ppy.sh/users/4384207) gab an, dass BWS "idealerweise keine dauerhafte Lösung des Problems" sei und plädierte stattdessen für eine Matchmaking-Rangliste (oder ähnliches), die vom [Spiel-Client](/wiki/Client) unterstützt wird.[^digitalhypno-discord-5]
+
+## Referenzen
+
+[^badge-appeals]: [Forumsnachricht "Badge Appeals"](https://osu.ppy.sh/community/forums/topics/1066357) von ::{ flag=US }:: [Kron05](https://osu.ppy.sh/users/10505107)
```
Comment (respektive) [inline wiki/Tournaments/Badge-weighted_seeding/de.md:53]:
```suggestion
[^badge-appeals]: [Forumsbeitrag "Badge Appeals"](https://osu.ppy.sh/community/forums/topics/1066357) von ::{ flag=US }:: [Kron05](https://osu.ppy.sh/users/10505107)
```

File: wiki/Tournaments/Badge-weighted_seeding/de.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+[^brtt-bws]: Abschnitt "BWS" aus der [Forumsnachricht *Baku's Random Team Tournament #3*](https://osu.ppy.sh/community/forums/topics/973512) von ::{ flag=DE }:: [Bakugo-](https://osu.ppy.sh/users/4990127)
```
Comment (respektive) [inline wiki/Tournaments/Badge-weighted_seeding/de.md:54]:
```suggestion
[^brtt-bws]: Abschnitt "BWS" aus dem [Forumsbeitrag *Baku's Random Team Tournament #3*](https://osu.ppy.sh/community/forums/topics/973512) von ::{ flag=DE }:: [Bakugo-](https://osu.ppy.sh/users/4990127)
```

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/Badge-weighted_seeding/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Tournaments/Badge-weighted_seeding/de.md:36]: This is better 👍

**Review by respektive — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**